docs/source/tracking.rst

```diff
@@ -934,6 +1029,45 @@ You can then pass authentication headers to MLflow using these :ref:`environment
 Additionally, you should ensure that the ``--backend-store-uri`` (which defaults to the
 ``./mlruns`` directory) points to a persistent (non-ephemeral) disk or database connection.
 
+.. _artifact_only_mode:
+
+Using the Tracking Server exclusively for proxy artifact access
+---------------------------------------------------------------
+
+To use an instance of the MLflow tracking server *exclusively* for artifact operations ( :ref:`scenario_6` ),
+start a server with the optional parameters ``--serve-artifacts`` to enable proxy artifact handling and ``--artifacts-only``
+for disabling all other functionality of the tracking server.
+
+To start the MLflow server in this restricted mode with a proxy to an HDFS location (as an example):
+
+.. code-block:: bash
+
+    mlflow server \
+        --host 0.0.0.0 \
+        --port 8885 \
+        --default-artifact-root hdfs://myhost:8887/mlprojects/models \
+        --serve-artifacts \
+        --artifacts-only
+
+Using this tracking server for any tasks aside from the aforementioned ``mlflow.<flavor>.log_model()``,
+``mlflow.<flavor>.load_model()`` and ``client.list_artifacts(<run_id>)`` will throw an ``MLflowException``.
```

Comment:
There are other methods used for logging artifacts, e.g. `mlflow.log_artifact()`. Perhaps we should update the phrasing to indicate that this isn't an exhaustive list of artifact-based APIs. This is also Python-specific.

Reply:
rephrased the entire paragraph.